first-time-setup.sh

```diff
@@ -12,7 +12,7 @@ if [[ $JURISDICTION_NAME ]] && [[ $REPOSITORY_URL ]]
 then continue=Y
 fi
 
-if [[ -d .git ]]
+if [[ $CI ]] && [ ! -d .git ]
```

Comment:
I don't understand the condition change: we now refuse to proceed if we're in CI and there is no Git history, which should never happen… but we also lose the refusal to proceed safety if we're not in CI and there is a Git history. Shouldn't this be “only if not in CI and there is a Git history”? 🤔 
```suggestion
if [[ ! $CI ]] && [[ -d .git ]]
```

Reply:
My mistake.